collectors/constants.py

```diff
@@ -1,3 +1,5 @@
 from osidb.helpers import get_env
 
+# todo?
```

Comment:
What is the meaning of this?

Reply:
I tried some `getenv` things here, but you can ignore it. It is mostly for visibility now.